README.md

```diff
@@ -20,7 +23,7 @@ git clone --recurse-submodules https://github.com/AlignmentResearch/go_attack.gi
 git clone --recurse-submodules git@github.com:AlignmentResearch/go_attack.git
 ```
 
-You can run `pip install -e .[dev]` inside the project root directory to install all necessary dependencies.
+Run `pip install -e .[dev]` inside the project root directory to install all necessary dependencies.
```

Comment:
Consider adding single quotes around `[` that can cause some shells to try and expand it as a glob
```suggestion
Run `pip install -e '.[dev]'` inside the project root directory to install all necessary dependencies.
```